client/src/component/header.css

```diff
@@ -0,0 +1,23 @@
+header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	width: 100%;
+	height: 40px;
+	background-color: #efefef;
+}
+.logo {
+	height: 40px;
+	margin-left: 10px;
+}
+.navbar {
+	display: flex;
+	flex-direction: row;
+	gap: 10px;
+	list-style-type: none;
+	margin-right: 10px;
+}
+.link {
```

Comment:
careful with name conflicts for classes like .link @MohammadSefidgar can explain BEM which he has been using which would give a classname like .header__link here